README.md

```
@@ -1,2 +1,135 @@
+
+## 🔖 ERD
+
+<img width="100%" alt="FISA" src="https://github.com/user-attachments/assets/f12ae76d-2c93-4d14-8ced-8c6e5e66b185" />
+
+---
+
+## 🗂 Repository 구조
+
+> 도메인 단위로 나누고, 각 도메인 하위에 `controller / service / repository / dto / entity`를 두는 형태
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)